firstvoices/backend/views/sites_views.py

```diff
@@ -32,18 +37,28 @@
             404: OpenApiResponse(description="Todo: Not Found"),
         },
     ),
+    update=extend_schema(
```

Comment:
Does the request format show properly in the docs?

Reply:
Yes, it shows as a 'PUT' request, with the correct request body and response structures.